### PR TITLE
Beginning of major redesign 

### PR DESCRIPTION
**Pull request by BenPearo**
Includes many unfinished pages, but has the architecture for the new clubs page and event cards. This branch will serve to let students contribute the the redesign without storing breaking changes on the main branch.

### CODE REVIEW

File: _layouts/club.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                    {% endif %}
+                    
+
+
+                    <!-- title -->
+                    <!-- tagline -->
+                    <!--    about -->
+                    <!-- custom -->
+                    <!-- socials -->
+                    <!-- pictures? -->
```
Comment (keeferrourke) [inline _layouts/club.html:72]: what's up with these comments?

File: _includes/header.html [hunk 2]
```diff
@@ -12,6 +12,7 @@
         </div>
         <div class="collapse navbar-collapse" id="main-navbar-collapse">
           <ul class="nav navbar-nav navbar-right">
+            <li><a href="/clubs/">Clubs</a></li>
```
Comment (keeferrourke) [inline _includes/header.html:15]: The plan is to turn this into a drop-down menu with options for each of the SOCCSOs, correct?

File: _posts/clubs/2021-03-28-socis.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    - {name: Mack Kean, position: VP Finance, email: vp-finance@socis.ca}
+    - {name: Zaza Hashmi, position: VP Communications, email: vp-communications@socis.ca}
+    - {name: Ryan Sheppard, position: VP Social, email: secretary@socis.ca}
+    - {name: Keefer Rourke, position: System Admin, email: admin@socis.ca}
+staff: 
+    - {name: Sara Adi, position: First Year Representative, email: first-year-rep@socis.ca}
+    - {name: Zayn Abbas, position: Second Year Representative, email: first-year-rep@socis.ca}
+    - {name: Sara Adi, position: First Year Representative, email: first-year-rep@socis.ca}
+    - {name: Sara Adi, position: First Year Representative, email: first-year-rep@socis.ca}
+---
```
Comment (keeferrourke) [inline _posts/clubs/2021-03-28-socis.md:20]: This is so clean! :heart:  :sparkles:

File: _posts/clubs/2021-03-28-socis.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    - {name: Kaan Sinar, position: VP External, email: vp-external@socis.ca}
+    - {name: Mack Kean, position: VP Finance, email: vp-finance@socis.ca}
+    - {name: Zaza Hashmi, position: VP Communications, email: vp-communications@socis.ca}
+    - {name: Ryan Sheppard, position: VP Social, email: secretary@socis.ca}
+    - {name: Keefer Rourke, position: System Admin, email: admin@socis.ca}
+staff: 
+    - {name: Sara Adi, position: First Year Representative, email: first-year-rep@socis.ca}
+    - {name: Zayn Abbas, position: Second Year Representative, email: first-year-rep@socis.ca}
+    - {name: Sara Adi, position: First Year Representative, email: first-year-rep@socis.ca}
+    - {name: Sara Adi, position: First Year Representative, email: first-year-rep@socis.ca}
```
Comment (keeferrourke) [inline _posts/clubs/2021-03-28-socis.md:19]: These emails are wrong. I think you should have:

- `first-year-rep@socis.ca`
- `student-rep2@socis.ca`
- `student-rep3@socis.ca`
- `student-rep4@socis.ca`

Comment (keeferrourke) [reply _posts/clubs/2021-03-28-socis.md:19]: Also Sara is here 3 times! @BenPearo put yourself and Miranda on lines 18 and 19.

**Review by keeferrourke — CHANGES_REQUESTED** [review]

File: clubs/index.html [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+layout: default
+---
+
+<div class="container">
```
Comment (keeferrourke) [inline clubs/index.html:5]: How do I preview this page?

visiting `localhost:4000/clubs` is a 404. Am I missing something?

File: _layouts/club.html [hunk 6]
```diff
@@ -0,0 +1,103 @@
+
+<!DOCTYPE html>
+<head>
+    <meta charset="utf-8">
+    <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
+    <meta content="SOCIS" name="author">
+    <meta content="Computer Science We are a group of computer science students for computer science students. We coordinate with the School of Computer Science in …" name="description">
```
[the remark this answers is not in the document]
Comment (BenPearo) [reply _layouts/club.html:7]: Good idea. This is on all our pages so I will do that

File: _posts/events/2021-02-10-prose-file.md [hunk 7]
```diff
@@ -0,0 +1,5 @@
+---
+published: true
+---
+## A New Post
+Wow prose is sweet
```
[the remark this answers is not in the document]
Comment (BenPearo) [reply _posts/events/2021-02-10-prose-file.md:5]: correct will remove

File: _includes/club-card.html [hunk 8]
```diff
@@ -0,0 +1,9 @@
+<article>
```
[the remark this answers is not in the document]
Comment (BenPearo) [reply _includes/club-card.html:1]: yep

File: _layouts/club.html (hunk 1 above)
Comment (BenPearo) [reply _layouts/club.html:72]: Oh I should remove those

File: _includes/header.html (hunk 2 above)
Comment (BenPearo) [reply _includes/header.html:15]: I was planning on making a sort of club homepage. A dropdown would mean we don't have to make cards for clubs which is an idea I like

File: _posts/clubs/2021-03-28-socis.md (hunk 4 above)
Comment (BenPearo) [reply _posts/clubs/2021-03-28-socis.md:19]: Thank you! I was waiting for confirmation in slack but sent it through

File: clubs/index.html (hunk 5 above)
Comment (BenPearo) [reply clubs/index.html:5]: The clubs tab at the top should navigate you there

File: _includes/header.html (hunk 2 above)
Comment (keeferrourke) [reply _includes/header.html:15]: Okay cool, a club homepage is better :)

File: clubs/index.html (hunk 5 above)
Comment (keeferrourke) [reply clubs/index.html:5]: Hm I don't see a clubs tab? 

image

image

**Review by BenPearo — COMMENTED** [review]
All requested changes have been made. Please test locally before approving.

File: clubs/index.html (hunk 5 above)
Comment (keeferrourke) [reply clubs/index.html:5]: Ha yeah my bad :sweat:

**Review by keeferrourke — APPROVED** [review]

**Pull request merged**